application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/config/repair/RepairConfig.java

```diff
@@ -144,6 +146,18 @@ public final void validate(final String repairConfigType)
         }
     }
 
+    @JsonProperty("initial_delay")
+    public final Interval getMyInitialDelay()
+    {
+        return myInitialDelay;
+    }
+
+    @JsonProperty("initial_delay")
+    public final void setMyInitialDelay(final Interval slideOff)
```

Comment:
Same here; don't name the method "my...."!. Also, "slideOff"? Why a completely new name? Why not just initialDelay, delay etc?